File: server/Models/GolfCourse.cs
using Microsoft.OpenApi.Models;
using Microsoft.Playwright;

namespace fairwayfinder.Models

{
  public class GolfCourse
  {
    public int Id { get; set; }
    public string Name { get; set; }
    public string Img { get; set; }
    public string Location { get; set; }
    public string BookingSoftware { get; set; }
    public string FetchUrl { get; set; }
  }
}

File: client/src/models/GolfCourse.js
export class GolfCourse {
  constructor(data) {
    this.id = data.id
    this.name = data.name
    this.img = data.img
    this.location = data.location
    this.bookingSoftware = data.bookingSoftware
    this.fetchUrl = data.fetchUrl
    this.bookingUrl = data.bookingUrl
  }
}